relayer-cli/src/conclude.rs

```diff
@@ -136,7 +136,6 @@ impl fmt::Display for Result {
 /// which typically signals a success (UNIX process return code `0`) or an error (code `1`). An
 /// optional `result` can be added to an output.
 ///
-#[derive(Debug)]
```

Comment:
Should there be an explicit `impl fmt::Debug for Output` now that the `derive` has been removed?

Reply:
Not sure we need the `Debug` impl here. I removed the `derive` since it was not being used for anything.

We can re-add the `derive(Debug)` later on if needed. But I would actually prefer to make `Output` work with `Display` instead of `Debug`. The reason is that with `Debug` the output is too verbose and it's difficult to parse the logs, so we're forced to override of `impl Debug for X` to make output more concise (like we do in this PR and in here https://github.com/informalsystems/ibc-rs/pull/1944). Going with `Display` instead of `Debug` will be better, but that would require some bigger refactoring (which we can do with #594 if we see the need).